### PR TITLE
Added git repo creation configuration with secrets section

### LINKED ISSUE #1623: Update Git repo creation docs to include example leveraging k8s secrets

### PR DESCRIPTION
**Pull request by DiegoWong**
Which other branches should this be merged with (if any)?

none

resolves #1623

### CODE REVIEW

**Comment by RichardJJG** [comment]
Thanks, @DiegoWong

File: tap-gui/plugins/application-accelerator-git-repo.hbs.md [hunk 1]
```diff
@@ -45,6 +45,31 @@ These steps describe an example configuration that uses GitHub:
...
+
+1. Create the kubernetes secret with the desired values
+
+```bash
+kubectl create secret githubOauthApp \
+--from-literal=clientSecret=GITHUB-CLIENT-SECRET \
+--from-literal=clientId=GITHUB-CLIENT-ID
+```
+
+2. Edit the `app-config.yaml` using the envitonment variables
```
Comment (matt-dres) [inline tap-gui/plugins/application-accelerator-git-repo.hbs.md:61]: "envitoment" -> "environment"

File: tap-gui/plugins/application-accelerator-git-repo.hbs.md [hunk 2]
```diff
@@ -45,6 +45,31 @@ These steps describe an example configuration that uses GitHub:
    For more information, see the
    [Backstage documentation](https://backstage.io/docs/integrations/github/locations).
 
+### Using kubernetes secrets
```
Comment (matt-dres) [inline tap-gui/plugins/application-accelerator-git-repo.hbs.md:48]: "kubernetes" should be capitalized

**Review by matt-dres — COMMENTED** [review]
LGTM 👍 Minor nitpicks

File: tap-gui/plugins/application-accelerator-git-repo.hbs.md (hunk 2 above)
Comment (RichardJJG) [reply tap-gui/plugins/application-accelerator-git-repo.hbs.md:48]: I'll fix this and any other writing errors I spot

**Pull request merged**